### DISCUSSION TITLE
StateContainer - Repeated count  missing

### DISCUSSION THREAD

**Discussion by gabsamples6** [CommunityToolkit/Maui#1125] [New Feature Discussions]
Hi,
wondering if the repeatedcount can be added to this control, it made such a difference when you loading a list - very common as you know.

thanks

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889

**Reply by gabsamples6** [reply]
@CliffAgius i didnt not watch the entire video but all I wanted to know if this has been added or not.

**Reply by CliffAgius** [reply]
Hi @gabsamples6 the repeated count is not part of the Toolkit at present and to be honest I am not 100% sure what your asking for, if you could elaborate a little even a sample would be great to help us understand.

The idea from the Stand-up is that as a small team we found that there were lots of ideas for features that were posted on the discussions board but we needed to find those the community really wanted rather than the wants of one developer. So for a discussion to get looked at by the team we now have a threshold of 6+ upvotes on the idea inside 6 months or it will be closed and as you can see your post has just your vote hence being closed.

However it could be that like myself others are not sure what your suggesting so maybe put a bit more into your suggestion with maybe an API around it and how it would work and be surfaced and then it may get the votes, or at the very least start a discussion.

I hope that makes sense and please don't think I am pushing the idea away we just needed to tidy up the repo to find what we should be working on. 

Let me know if you want this re-opened so you can add on.

**Reply by gabsamples6** [reply]
Hi there,
firstly I dont know how you do it a pilot and a dev.. WOW.. I can hardly be a dev with family and all that. 

Now not sure if you remember this guy https://thewissen.io/  is the actual original code writer of this control now renamed StateContainer.. 

What was cool about it is that lets suppose you were doing some "shimmering" and loading a list -  instead of having 3 frame/borders with some fake lines in it you could set - repeatcount=3 and it would make up those .. 

total regression in that sense.
hope makes sense

**Reply by CliffAgius** [reply]
I do remember Stephens StateContainer control I used it a few times and I think get what your suggesting.

To take this forward I think we would just need to add some details to this discussion explaining what your thinking and possibly add a rough outline of the API and how it would be used.  From that it can be shared and hopefully get the community engaged, without this the very small team can't in all honesty work on the idea as it could be just one person wanting the feature hence the new rules.
